src/libnvme.map

```diff
@@ -20,6 +20,8 @@ LIBNVME_1_5 {
 		nvme_root_set_application;
 		nvme_subsystem_get_application;
 		nvme_subsystem_set_application;
+		nvme_set_debug;
+		nvme_get_debug;
```

Comment:
These two entries should be under the 1.6 section as we already released 1.5.

Reply:
Fixed the version.